src/kbmod/data_interface.py

```diff
@@ -0,0 +1,150 @@
+import multiprocessing as mp
+import os
+import time
+
+from astropy.io import fits
+from astropy.wcs import WCS
+import numpy as np
+from scipy.special import erfinv  # import mpmath
+
+import kbmod.search as kb
+
+from .file_utils import *
+from .filters.clustering_filters import DBSCANFilter
+from .filters.stats_filters import *
+from .result_list import ResultList, ResultRow
+
+
+class Interface:
```

Comment:
Long term question (no changed needed in this PR): Do we need `Interface` to be a class? Or should it be a few static methods?

Reply:
I agree, it would probably be better to just have them be static!